packages/happn-3/lib/client.js

```diff
@@ -538,7 +538,7 @@
     };
   };
 
-  HappnClient.prototype.getScript = function (url, callback) {
+  HappnClient.prototype.getScript = utils.maybePromisify(function (url, callback) {
```

Comment:
Is there a reason these were wrapped? they are actually internal methods (so not part of the client API per se) - my next job is going to be rationalizing this client code - as it is very old...

Reply:
I needed to wrap revoke-token.  so thought do all functions on the prototype that take a callback